content/download/index.html.haml

```diff
@@ -163,6 +163,10 @@ title: Jenkins installation and setup
                 %span.icon
                 %span.title
                   Ubuntu/Debian
+              %a.list-group-item.list-group-item-action{:href=>'http://pkg.openindiana.org/hipster/en/search.shtml%3Ftoken=jenkins&amp;action=Search'}
```

Comment:
The problem is that this line has a tab instead of a space. HAML is like Python in that it's white-space sensitive